src/NServiceBus.Transport.RabbitMQ/Receiving/ModelExtensions.cs

```diff
@@ -1,46 +1,25 @@
 ﻿namespace NServiceBus.Transport.RabbitMQ
 {
-    using System.Threading.Tasks;
```

Comment:
The tasks used in this class are redundant.

At one time, the methods accepted a `TaskScheduler`, but [when that parameter was removed]((https://github.com/Particular/NServiceBus.RabbitMQ/commit/29080f305e7cf77c119da0c50e87060fb04a8db5#diff-963cdc5b2f2a1f7238c343f73499f46db12b5ba32483103a6e95dc50ce3c76e0)) in favour of `TaskScheduler.Default`, they became redundant. I mentioned this to @bording and he concurs.

(The use of `CancellationToken.None` in `TaskEx` drew my attention to this.)